resources/scripts.js

```diff
@@ -402,7 +402,14 @@ function displayMessage(str) {
 
 function boardClick(e) {
     let numClicked = parseInt(e.target.alt);
-    const PLACENOPOINT = `You cannot place on ${numClicked} because no point is open`
+    const PLACENOPOINT = `You cannot place on ${numClicked} because no point is open`;
+    const PLACEAGAIN = `You have already made a bet on ${numClicked}`;
+    for (let i = 0; i < secondaryBetList.length; i++) {
```

Comment:
Is there a way to check this without a loop?

Reply:
Yes! Use an object

Reply:
Object implemented instead of array/object.